schema/template.schema.json

```diff
@@ -1,8 +1,14 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://adobe.io/schemas/aio/project",
+  "$id": "https://adobe.io/schemas/app-builder-templates/installation-schema/1",
   "type": "object",
   "properties": {
+    "$schema": {
+      "$ref": "#/definitions/$schema"
+    },
+    "$id": {
+      "$ref": "#/definitions/$id"
+    },
```

Comment:
https://wiki.corp.adobe.com/display/AdobeCloudPlatform/App+Builder+Template+Install+Configuration requires `$schema` and `$id` for versioning.